scripts/mitm.py

```diff
@@ -95,7 +94,10 @@ def my_callback(coJeu, coSer):
         target = dofus.pid
 
     if args.attach:
-        target = args.pid or "dofus"
+        if sys.platform == "linux" or sys.platform == "win32":
```

Comment:
I dont think it's "Dofus.exe" on linux

Reply:
Yes it is. On linux, Dofus.exe is executed by Wine